versioned/tiered/dynamodb/src/test/java/com/dremio/nessie/versioned/impl/condition/TestExpressions.java

```diff
@@ -115,14 +115,13 @@ void updateAddClause() {
   void updateSetClause() {
     UpdateExpression e0 = ImmutableUpdateExpression.builder().addClauses(
         SetClause.equals(p0, av0),
-        SetClause.ifNotExists(p1, p0, av1),
         SetClause.appendToList(p1, Entity.ofList(av1))
         ).build();
     AliasCollectorImpl c = new AliasCollectorImpl();
     UpdateExpression e0p = e0.alias(c);
-    assertEquals(" SET p0 = :v0, p1 = if_not_exists(p0, :v1), p1 = list_append(p1, :v2)", e0p.toUpdateExpressionString());
+    assertEquals(" SET p0 = :v0, p1 = list_append(p1, :v1)", e0p.toUpdateExpressionString());
```

Comment:
Why are we removing `if_not_exists` for the dynamo tests? I believe this was required by dynamo under certain circumstances.